tests/sentry/api/endpoints/test_project_details.py

```diff
@@ -1325,6 +1325,12 @@ def test_symbol_sources_deletion(self):
             assert resp.status_code == 200
             assert project.get_option("sentry:symbol_sources", orjson.dumps([source1]).decode())
 
+    @with_feature("organizations:metrics-extrapolation")
+    def test_extrapolate_metrics(self):
+        resp = self.get_success_response(self.org_slug, self.proj_slug, extrapolateMetrics=False)
+        assert self.project.get_option("sentry:extrapolate_metrics") is False
+        assert resp.data["extrapolateMetrics"] is False
```

Comment:
Can you please below this, in a same test case, add a request where you also update `extrapolateMetrics` to `True` and checks that everything is updated as expected. Because default value for `extrapolateMetrics` is False and there might be the situation in the future when something will change and update won't work anymore, but the test won't cover it